apps/condo/domains/organization/schema/FindOrganizationsByTinLog.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Generated by `createschema organization.FindOrganizationsByTinLog 'user:Relationship:User:SET_NULL; tin:Text'`
+ */
+
+const get = require('lodash/get')
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { versioned, uuided, tracked, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/organization/access/FindOrganizationsByTinLog')
+
+
+const ERRORS = {
+    USER_IS_MISSING: {
+        code: BAD_USER_INPUT,
+        type: 'USER_IS_MISSING',
+        variable: ['data', 'user'],
+        message: 'The "user" field must be specified',
+    },
+}
+
+const FindOrganizationsByTinLog = new GQLListSchema('FindOrganizationsByTinLog', {
+    schemaDoc: 'Logging the query "FindOrganizationsByTin"',
+    fields: {
+
+        user: {
+            schemaDoc: 'The user who caused the query',
+            type: 'Relationship',
+            ref: 'User',
+            isRequired: true,
+            kmigratorOptions: { null: true, on_delete: 'models.SET_NULL' },
+        },
+
+        userId: {
```

Comment:
In case of hard deletion of User it will be hard for us to track down his phone / email / identity 

Do you think userId is sufficient here? Maybe we can just freeze user object and store it here?

Just a nitpick. Probably we do not want it, because of GDPR ...

Reply:
I remembered that we do not delete users, we only reset them. If we set `deletedAt` - this means that we banned the user and in this case in a admin panel we will not see a link to the user, but we will be able to see the userId

So I think we can remove this field